src/components/ui/avatar/Avatar.tsx

```diff
@@ -0,0 +1,38 @@
+import { Avatar as ArkAvatar, type AvatarProps as ArkAvatarProps } from '@ark-ui/react/avatar';
+import { forwardRef } from 'react';
+import { avatar, type AvatarVariantProps } from '@/styled/recipes';
+
+export interface AvatarProps extends ArkAvatarProps, AvatarVariantProps {
+  name?: string;
+  src?: string;
+}
+
+export const Avatar = forwardRef<HTMLDivElement, AvatarProps>((props, ref) => {
+  const [variantProps, localProps] = avatar.splitVariantProps(props);
+  const { name, src, ...rootProps } = localProps;
+  const styles = avatar(variantProps);
+
+  return (
+    <ArkAvatar.Root ref={ref} className={styles.root} {...rootProps}>
+      <ArkAvatar.Fallback className={styles.fallback}>{getInitials(name) || <UserIcon />}</ArkAvatar.Fallback>
+      <ArkAvatar.Image className={styles.image} src={src} alt={name} />
+    </ArkAvatar.Root>
+  );
+});
+
+Avatar.displayName = 'Avatar';
+
+const UserIcon = () => (
```

Comment:
You should we will create 1 file manage Icons.
Example

Import {UserIcon} from 'icons'.